.travis.yml

```diff
@@ -9,6 +9,11 @@ before_script:
 node_js:
 - 'stable'
 
+env:
```

Comment:
You can remove the environment variables. It has been added on Travis

Reply:
Implemented